docs/standard/microservices-architecture/implement-resilient-applications/implement-circuit-breaker-pattern.md

```diff
@@ -1,177 +1,107 @@
-        var token = await context.Authentication.GetTokenAsync("access_token");
-        _apiClient.Inst.DefaultRequestHeaders.Authorization = new
-            System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", token);
-        _apiClient.Inst.DefaultRequestHeaders.Add("x-requestid",
-            order.RequestId.ToString());
-        var ordersUrl = $"{_remoteServiceBaseUrl}/new";
-        order.CardTypeId = 1;
-        order.CardExpirationApiFormat();
-        SetFakeIdToProducts(order);
-        var response = await _apiClient.PostAsync(ordersUrl, order);
-        response.EnsureSuccessStatusCode();
-    }
+    return HttpPolicyExtensions
+        .HandleTransientHttpError()
+        .CircuitBreakerAsync(5, TimeSpan.FromSeconds(30));
 }
 ```
 
-Whenever the \_apiClient member object is used, it internally uses the wrapper class with Polly policiesؙ—the Retry policy, the Circuit Breaker policy, and any other policy that you might want to apply from the Polly policies collection.
+In the code example above, the circuit breaker policy is configured so it break or open the circuit when there have been five exceptions when retrying the Http requests. Then, 30 seconds will be the duration or the break.
 
-## Testing retries in eShopOnContainers
+Circuit breakers should also be used to redirect requests to a fallback infrastructure if you had issues in a particular resource that is deployed in a different environment than the client application or service that is performing the HTTP call. That way, if there is an outage in the datacenter that impacts only your backend microservices but not your client applications, the client applications can redirect to the fallback services. Polly is planning a new policy to automate this [failover policy](https://github.com/App-vNext/Polly/wiki/Polly-Roadmap#failover-policy) scenario. 
 
-Whenever you start the eShopOnContainers solution in a Docker host, it needs to start multiple containers. Some of the containers are slower to start and initialize, like the SQL Server container. This is especially true the first time you deploy the eShopOnContainers application into Docker, because it needs to set up the images and the database. The fact that some containers start slower than others can cause the rest of the services to initially throw HTTP exceptions, even if you set dependencies between containers at the docker-compose level, as explained in previous sections. Those docker-compose dependencies between containers are just at the process level. The container’s entry point process might be started, but SQL Server might not be ready for queries. The result can be a cascade of errors, and the application can get an exception when trying to consume that particular container.
+Of course, all those features are for cases where you are managing the failover from within the .NET code, as opposed to having it managed automatically for you by Azure, with location transparency. 
```

Comment:
Remove of course since it might not be obvious for the reader 

you are managing -> you're managing

Reply:
👍